### PR TITLE
Notification UX tweaks

### PR DESCRIPTION
**Pull request by faissalMT**
deepinscreenshot_select-area_20181024142148

### CODE REVIEW

File: lib/delivery_mechanism/web_routes.rb [hunk 1]
```diff
@@ -83,10 +83,15 @@ class WebRoutes < Sinatra::Base
...
       guard_access env, params, request do |request_hash|
+        begin
         DeliveryMechanism::Controllers::PostSubmitReturn.new(
           submit_return: @dependency_factory.get_use_case(:submit_return),
-          notify_project_members: @dependency_factory.get_use_case(:notify_project_members)
-        ).execute(request, request_hash, response)
+          notify_project_members: @dependency_factory.get_use_case(:notify_project_members),
+          check_api_key: @dependency_factory.get_use_case(:check_api_key)
+        ).execute(env, request, request_hash, response)
+        rescue Exception => e
```
Comment (danielburnley) [inline lib/delivery_mechanism/web_routes.rb]: I assume this was debugging code that was left in?

Comment (faissalMT) [reply lib/delivery_mechanism/web_routes.rb]: Good spot

File: spec/unit/local_authority/use_case/create_api_key_spec.rb [hunk 2]
```diff
@@ -25,12 +25,15 @@
       )
 
       expect(decoded_key[0]['project_id']).to eq(1)
+      expect(decoded_key[0]['email']).to eq('cat@cathouse.com')
     end
 
+    #it 'stories the email within the api key'
```
Comment (danielburnley) [inline spec/unit/local_authority/use_case/create_api_key_spec.rb]: :eyes:

Comment (faissalMT) [reply spec/unit/local_authority/use_case/create_api_key_spec.rb]: Forgot to remove that, ended up laying out the tests differently.

**Pull request merged**